config/config.php

```diff
@@ -6,14 +6,14 @@
 $configManager = new ConfigAggregator([
 
     // App packages
-    \Web\ConfigProvider::class,
     \Category\ConfigProvider::class,
     \Article\ConfigProvider::class,
     \Menu\ConfigProvider::class,
     \Admin\ConfigProvider::class,
     \Newsletter\ConfigProvider::class,
     \Page\ConfigProvider::class,
     \Meetup\ConfigProvider::class,
+    \Web\ConfigProvider::class,
```

Comment:
Web  package need to be first one because of router. Routes are found from the bottom, so last would be Web's routes. In Web package there are "magic" routes to catch everything like  /:url_slug 
that means if we later have route /contact-us  it will not be triggered.